.travis.yml

```diff
@@ -22,8 +22,8 @@ env:
     - CMAKE_PREFIX_PATH=${INSTALL_PREFIX}
     - LIBSODIUM_TAG=1.0.11
     - LIBSODIUM_DIR=${TRAVIS_BUILD_DIR}/libsodium
-    - AMCL_TAG=4.7.0
-    - AMCL_DIR=${TRAVIS_BUILD_DIR}/milagro-crypto-c/install
+    - AMCL_TAG=4.7.3
```

Comment:
Was there a specific reason to bump from `4.7.0` to `4.7.3`? 

It's almost certainly fine, but I think it's best to keep it at `4.7.0`, since that's currently what version the deb on our bintray is

Reply:
Only because that was the release version when I took the .tar.gz from our AMCL fork